buildSrc/src/main/java/dependencies/Versions.kt

```diff
@@ -7,7 +7,7 @@ object Versions {
 
     const val androidx_fragment = "1.4.1"
     const val dagger = "2.42"
-    const val espresso = "3.4.0"
+    const val espresso = "3.5.0-beta01"
```

Comment:
This is still beta, but the releases for Espresso move slowly, and we've been having enough trouble that I think it's worth it.